<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1398</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dataset Tools</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="1">
    <widget class="QLineEdit" name="fileFilterLed"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Caption Filter</string>
     </property>
    </widget>
   </item>
   <item row="14" column="8">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="includeSubdirCbx">
     <property name="toolTip">
      <string>Include subdirectories when loading images</string>
     </property>
     <property name="text">
      <string>Include Subdirectories</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="browseBtn">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QComboBox" name="fileFilterCmb"/>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="openBtn">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="captionFilterLed"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>File/Path Filter</string>
     </property>
    </widget>
   </item>
   <item row="4" column="7" rowspan="10" colspan="5">
    <layout class="QVBoxLayout" name="canvasLay">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="7" column="2">
    <widget class="QComboBox" name="captionFilterCmb"/>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QCheckBox" name="maskFilterCbx">
     <property name="text">
      <string>With Mask Only</string>
     </property>
    </widget>
   </item>
   <item row="15" column="7" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPlainTextEdit" name="captionTed">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::LineWrapMode::NoWrap</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="saveCaptionBtn">
         <property name="text">
          <string>Save Caption</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteCaptionBtn">
         <property name="text">
          <string>Delete Caption</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetCaptionBtn">
         <property name="text">
          <string>Reset Caption</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="10" column="0" rowspan="6" colspan="3">
    <widget class="QTreeWidget" name="fileTreeWdg">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::CurrentChanged</set>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLabel" name="numFilesLbl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QCheckBox" name="captionFilterCbx">
     <property name="text">
      <string>With Caption Only</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>openBtn</tabstop>
  <tabstop>includeSubdirCbx</tabstop>
  <tabstop>browseBtn</tabstop>
  <tabstop>fileFilterLed</tabstop>
  <tabstop>fileFilterCmb</tabstop>
  <tabstop>captionFilterLed</tabstop>
  <tabstop>captionFilterCmb</tabstop>
  <tabstop>maskFilterCbx</tabstop>
  <tabstop>captionFilterCbx</tabstop>
  <tabstop>fileTreeWdg</tabstop>
  <tabstop>captionTed</tabstop>
  <tabstop>saveCaptionBtn</tabstop>
  <tabstop>deleteCaptionBtn</tabstop>
  <tabstop>resetCaptionBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
